cmd/server.go

```diff
@@ -83,6 +83,11 @@ func serverDelete(ctx *cli.Context) error {
 		return err
 	}
 
+	if len(cf.Servers) == 0 {
+		fmt.Println("No servers are currently configured.")
+		return nil
+	}
```

Comment:
```suggestion
	if len(cf.Servers) == 0 {
		return errors.New("no servers are currently configured")
	}
```

Reply:
Good catch, @jakefhyde.

If we're doing this, I'd say it puts more weight into separating it into a func and doing something like below in all 3 places:
```
	if err := validateServersConfig(cf); err != nil {
		return err
	}
```

Reply:
If we return an error the resulting log message is `FATA[0000] No servers are currently configured. `. I'm not sure if we want to include `FATA[0000]` when this occurs, as it's not necessarily a fatal _error_ and does not add any additional information to the person running the command. Adding the `FATA[0000]` is done by `github.com/urfave/cli`, so I cannot suppress that section of the log message if we do return an error. 

Do we still want to return an error in this case? Not have any servers configured isn't necessarily an error, although I do understand the sentiment that because this command did not result in the described action it could be considered an error. I'm fine with either decision, but I wanted to note the differences

Reply:
We already return errors from this func, though these errors have very low probability of actually happening.
Regardless, I agree with Jake that in case of no servers configured, we have error condition and CLI should not exit with 0.